lib/utils/BvEvent.js

```diff
@@ -23,7 +23,9 @@ export class BvEvent {
         let defaultPrevented = false;
         // Recreate preventDefault method. One way setter.
         this.preventDefault = function preventDefault() {
-            defaultPrevented = true;
+            if (this.cancelable) {
```

Comment:
Possibly. I figured we would be using this mostly in circumstances where we need the `preventDefault` capabilities, so those are inherently cancelable UI events. Then it would be easy to forget to pass `cancelable: true`. Not sure until we use it I guess.